### PR TITLE
Tfx

### PR DESCRIPTION
**Pull request by Mageswaran1989**

### CODE REVIEW

File: vitaflow/playground/tfx_east/README.md [hunk 1]
```diff
@@ -5,25 +5,112 @@
...
 
 ```
+    # Pyhton 2.7
+    cd
+    virtualenv -p python2.7 tfx-env
+    source ~/tfx-env/bin/activate
+    mkdir tfx; cd tfx
+    
+    pip install tensorflow==1.13.1
+    pip install tfx==0.12.0
```
Comment (reddyanilkumar) [inline vitaflow/playground/tfx_east/README.md]: Need to add manual installation from git for imports which we are using in the code.

File: vitaflow/playground/tfx_east/README.md [hunk 2]
```diff
@@ -5,25 +5,112 @@
...
+host    all             all             0.0.0.0/0               md5
+
+sudo vim /etc/postgresql/10/main/postgresql.conf
+listen_addresses = '*'
+
+sudo service postgresql restart
+
+#posgres url 
+postgresql://airflow:airflow@localhost:5432/airflow
+
```
Comment (reddyanilkumar) [inline vitaflow/playground/tfx_east/README.md]: In which file we update these parameters.?

Comment (Mageswaran1989) [reply vitaflow/playground/tfx_east/README.md]: this is general url schema, other configs are in $AIRFLOW_HOME/airflow.cfg

Comment (Mageswaran1989) [reply vitaflow/playground/tfx_east/README.md]: there is a link in the readme for reference

File: vitaflow/playground/tfx_east/README.md (hunk 1 above)
Comment (Mageswaran1989) [reply vitaflow/playground/tfx_east/README.md]: I have added this as part of Python 3 setup

**Comment by Mageswaran1989** [comment]
@reddyanilkumar review plz....

**Pull request merged**